kibble/api/translations.go

```diff
@@ -30,6 +30,7 @@ func LoadAllTranslations(cfg *models.Config, site *models.Site) error {
 
 	data, err := Get(cfg, path)
 	if err != nil {
+		log.Errorf("Translations from API failed to load %s", err)
```

Comment:
Isn't this log redundant given that the error here bubbles up and gets logged anyway?

Reply:
Yes, but I put this here because I wanted to discuss whether or not the process should be killed here 🤷‍♂️

Reply:
Just updated the error message to be clearer, still bubbles up, and removed log